ci/build_container/build_container_ubuntu.sh

```diff
@@ -20,6 +20,12 @@ apt-get update
 apt-get install -y bazel
 rm -rf /var/lib/apt/lists/*
 
+# Setup tcpdump for non-root.
+groupadd pcap
```

Comment:
shouldn't we update the build image, land the PR, wait for our build image to update, and then use it in a follow-up PR?

In related news
alyssar@alyssar02:~/git/envoy$ fix_format
WARNING: IPv4 forwarding is disabled. Networking will not work.
usermod: group 'pcap' does not exist

I guess it'll fix itself in about an hour?

Reply:
@alyssawilk as soon as https://github.com/envoyproxy/envoy/pull/4849 sanity will be restored. My bad.